### DISCUSSION TITLE
Apparent search bug

### DISCUSSION THREAD

**Discussion by goldfarbneal** [freelawproject/courtlistener#4038] [Q&A]
I run a search several times for "ordinary meaning," with the date restricted to before 1808. Each time, the results include cases that include "ordinary means" but not "ordinary meaning."

Why is that happening, and is there something I can do to prevent it?

**Comment by mlissner** [comment]
Hi @goldfarbneal, can you share the link, please?

**Comment by goldfarbneal** [comment]
https://www.courtlistener.com/?type=o&q=%22ordinary%20meaning%22&type=o&order_by=score%20desc&stat_Precedential=on&filed_before=01%2F01%2F1808

**Reply by mlissner** [reply]
Got it, thanks. So we are in the middle of revamping our search engine and it looks like you are opted into our old one that does stemming. That means that the ends of words are removed in clever ways to make them more likely to be the same word, so means == meaning to the old search engine.

The new search engine does this too, but it can be disabled by putting the query into quotes, as you've done. 

I think you can enable the new search engine for yourself by logging in, but I might be mistaken about that. 

In any case, when I do your query (on the new search engine), I get no results. If I log out, I see the problem you do.
